test/utils/yaml.test.ts

```diff
@@ -38,39 +38,8 @@ async function getPathInFile(
   );
 }
 
-function isWindows() {
-  // win32 applies to x64 arch too, is the platform name
-  return process.platform === 'win32';
-}
-
```

Comment:
IMHO, we should keep this utility function to avoid undigging it in the future.

We should also keep the list of brokenTest as empty and with a commented line explaining that the two arguments are the test name and link to bug.

If we revert all this code someone else will spend extra time trying to rediscover a way to reimplement xfail for mocha.

Reply:
I'd rather not maintain this.

Reply:
I tend to agree with @ssbarnea since we are not actively working/testing with windows WSL we will this login in future or in cases were the test are flaky and we don't want to block the CI for those tests.

Reply:
Also I don't see any substantial maintenance overhead for this piece of code.